source/Calamari.AzureAppService.Tests/RandomAzureRegion.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Linq;
+using JetBrains.Annotations;
+
+namespace Calamari.AzureAppService.Tests
+{
+    public static class RandomAzureRegion
+    {
+        static Random random = new Random();
+
+        static string[] regions = new[]
+        {
+            "southeastasia",
+            "centralus",
+            "eastus",
+            "eastus2",
+            "westus",
+            "westus2",
+            "westus3",
+            "australiaeast"
+        };
+
+        public static string GetRandomRegion(params string[] excludedRegions)
```

Comment:
Take it or leave it - but on first read of the calls to this method I assumed it was choosing a random region from the regions provided as params.
```suggestion
        public static string GetRandomRegionExcluding(params string[] excludedRegions)
```